docs/spelling_wordlist

```diff
@@ -342,6 +342,7 @@ lowercasing
 lt
 lte
 Luhn
+LZMA
```

Comment:
Looks like this should be lowercase (from the CI failure)

Reply:
IMO it should be uppercase in the release notes, since `LZMA` is a shortcut from names "_Lempel-Ziv-Markov algorithm_" :smile: (see https://docs.python.org/3/library/lzma.html).

Reply:
Or that way, yes. 😀

Reply:
Yeah, I started thinking one way, then the other. Started out lowercased because of https://docs.python.org/3/library/tarfile.html

Let me know what we would prefer...

Reply:
Let's go with @felixxm said.

Reply:
Am referring to this now as an XZ archive -- `.tar.lzma` is considered legacy and `.tar.xz` uses xz-utils which supports the legacy format and the newer LZMA2...